packages/frontend-api/src/Model/Store/StoresFilterOptions.php

```diff
@@ -8,9 +8,11 @@ class StoresFilterOptions
 {
     /**
      * @param string|null $searchText
+     * @param array{latitude: string, longitude: string}|null $coordinates
      */
     public function __construct(
         protected readonly ?string $searchText,
+        protected readonly ?array $coordinates = null,
```

Comment:
nitpick, I would prefer an object with a factory for coordinates

Reply:
I was thinking about it for a while. I became with some points when I would go with Object:

1. If there would be some more use cases for using Coordinates
2. If we somehow managed to automatically create that object in the query instead of manually creating it

Anyway I think that object should be final as its basic type :) 

Do you have some ideas? :)

Reply:
I can live with array :slightly_smiling_face: